src/topology/config/mod.rs

```diff
@@ -26,6 +26,8 @@ pub struct Config {
 pub struct GlobalOptions {
     #[serde(default = "default_data_dir")]
     pub data_dir: Option<PathBuf>,
+    #[serde(default)]
+    pub dns: Vec<String>,
```

Comment:
Would a better name be `dns_servers`? Might be a bit more clear.

Reply:
True